src/test/java/com/hocs/test/glue/decs/SearchStepDefs.java

```diff
@@ -132,33 +220,9 @@ public void theCreatedDCUCaseShouldBeVisibleInTheSearchResults() {
         search.assertCurrentCaseIsDisplayed();
     }
 
-    @And("I enter {string} into the {string} MPAM search criteria")
-    public void searchForMPAMCaseWith(String infoValue, String infoType) {
-        if (search.mpamCaseCheckbox.isCurrentlyVisible()) {
-            safeClickOn(search.mpamCaseCheckbox);
-        }
-        if (search.mtsCaseCheckbox.isCurrentlyVisible()) {
-            safeClickOn(search.mtsCaseCheckbox);
-        }
-        search.enterMPAMSearchCriteria(infoType, infoValue);
-        setSessionVariable("infoValue").to(infoValue);
-        setSessionVariable("infoType").to(infoType);
-    }
-
-    @And("I check that the MPAM search results have the correct {string}")
-    public void checkMPAMCaseHasCorrect(String infoType) {
-        search.assertMPAMInformationRandomSearchResult(infoType);
-    }
-
-    @And("I search for a case by it's case reference")
-    public void searchForCaseByReference() {
-        String caseRef = getCurrentCaseReference();
-        searchForMPAMCaseWith(caseRef, "Case Reference");
-        safeClickOn(searchButton);
-    }
-
     @And("the created case should be the only case visible in the search results")
     public void createdCaseShouldBeVisibleInTheSearchResults(){
+        String caseType = getCurrentCaseReference().split("/")[0];
```

Comment:
This is highlighted to me that we dont have a getCurrentCaseType method in basepage. Do you think we should create one that returns the caseType session variable?